pom.xml

```diff
@@ -13,7 +13,7 @@
   <name>ohsome API</name>
   <description>A public Web-RESTful-API for "ohsome" OpenStreetMap history data.</description>
   <packaging>jar</packaging>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>1.1.0</version>
```

Comment:
this we normally only change in the release branch, not in master.

Reply:
so to which version would it then change here?

Reply:
something like 1.2.0-SNAPSHOT?